### DISCUSSION TITLE
How to check Storage metric? Include index file, data file etc....

### DISCUSSION THREAD

**Discussion by hjlee9182** [milvus-io/milvus#30542] [Q&A and General discussion] [answered]
I want to know how much storage is currently occupied by Grafana. I couldn't find such information [here](https://milvus.io/docs/metrics_dashboard.md), so how can I know? If possible, it would be nice if each file (index file, data file, etc.) could be distinguished.

**Comment by yhmo** [comment, marked as answer]
In section "Data coordinator", the "Stored Binlog Size" is the the total size of stored data files. We don't have metric for single file respectively.
